markdown/solace-k8s-operator/solace-k8s-operator.md

```diff
@@ -0,0 +1,214 @@
+author: Ken Barr
+summary: How to build a Solace PubSubPlus Kubernetes Operator
+id: solace-k8s-operator
+tags: iguide
+categories: Kubernetes
+environments: Web
+status: Draft
+feedback link:
+analytics account: UA-3921398-10
+
+# How to build a Solace PubSubPlus Kubernetes Operator
+
+## Overview
+The kubernetes [Operator Pattern](https://kubernetes.io/docs/concepts/extend-kubernetes/operator) is exactly what it sounds like.  It is code or automation that acts like the set of runbooks a human operator of an application would use to deploy, monitor, troubleshoot, upgrade and decommission an application.  The operator can do these functions because it has a deep understanding of the application it is operating on and knowns if the application is functioning correctly. 
```

Comment:
Duration is missing under each sub section `Duration: 0:00:00` reflecting the estimated time needed to finsish the respected section. This will be used to show the total time remaining, so for example
![Screen Shot 2020-05-12 at 4 52 10 PM](https://user-images.githubusercontent.com/6826053/81744388-261ef600-9471-11ea-8a9b-d921e25b50fb.png)

So it'll be something like this to show the section will take 5 minutes to read
```
## Overview
Duration: 0:05:00

```